packages/search-input/src/SearchResultGroup/SearchResultGroup.types.ts

```diff
@@ -0,0 +1,13 @@
+import { PropsWithChildren, ReactNode } from 'react';
+
+export type SearchResultGroupProps = PropsWithChildren<{
```

Comment:
Do you have to use `PropsWithChildren` since `children` is required below?

Reply:
Nope, good catch 👍